src/shared-components/dropdown/index.tsx

```diff
@@ -30,9 +29,9 @@ type DropdownProps = {
   optionsContainerMaxHeight?: string;
   textAlign?: 'left' | 'center';
   /**
-   * The currently selected option. Can mount as `null`
+   * The currently selected option. Can be initially `undefined`
```

Comment:
Previous versions showed that anything `undefined` would be pre-selected, so I think that we might want to re-word this. I'm just not sure what `initially` undefined means, and what the use case would be of something going from initially undefined to some truthy value.